src/cryptonote_basic/miner.cpp

```diff
@@ -835,6 +835,12 @@ namespace cryptonote
 
     #elif defined(__APPLE__)
         
+      if(&IOPSGetTimeRemainingEstimate == NULL)
```

Comment:
I've actually changed this to use the availability macros and target conditionals which then allows for iOS builds (if ever needed!)